apps/api/src/order/order.controller.ts

```diff
@@ -17,6 +21,17 @@ export class OrderController {
     private jwtService: JwtService,
   ) {}
 
+  @Get('get/:id')
```

Comment:
no need to start the url with 'get'. 

utilize the id from the request payload. ask Ms. @TreshAnn